node_package/src/reactHydrateOrRender.ts

```
@@ -1,43 +1,47 @@
+  ? reactDomClient.hydrateRoot
+  : (domNode, reactElement) => ReactDOM.hydrate(reactElement, domNode);
+
+export function reactRender(domNode: Element, reactElement: ReactElement): RenderReturnType {
+  if (supportsRootApi) {
+    const root = reactDomClient.createRoot(domNode);
+    root.render(reactElement);
+    return root;
+  }
+
+  // eslint-disable-next-line react/no-render-return-value
+  return ReactDOM.render(reactElement, domNode);
```

Biome (1.9.4)

[error] Do not depend on the value returned by the function ReactDOM.render().

The returned value is legacy and future versions of React might return that value asynchronously.
Check the React documentation for more information.

(lint/correctness/noRenderReturnValue)